src/client/components/MyProfile/YourUploadedCVs/RatingStars.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Star } from './Star';
+
+export function RatingStars({ averageStars }) {
+  const reminder = `${((averageStars % 20) / 20) * 100}%`;
```

Comment:
Is this correct?

```suggestion
  const emptyStars= `${((averageStars % 20) / 20) * 100}%`;
```

Reply:
This is not emtyStars. I just fet average stars and change the number to two parts. one part is integer and one one prt is not integer. for example if the average number is 3.72 so the inetegr part is 3 and the decimal part is 0.72. I put decimal part in reminder and integr part in filledStar. and all those works are because of spacing between stars in design. I did not ny solution otherwise this way. it there was not any distance between stars I could code the starts with combioning content and :before and just I could define with based on the average. But we can not do that here. and that is why I coose this solution.